### DISCUSSION TITLE
Raw logs are publicly accessible

### DISCUSSION THREAD

**Discussion by VoidMonk** [community/community#25922] [Actions]
<p>In my opinion, raw logs or any other GitHub Action output within a private repo should be private as well.</p>

**Comment by jupe** [comment]
<p>Is this really true ?</p>

**Comment by chrispat** [comment, marked as answer]
<p>The raw logs for private are not publicly accessible.  You must have access to the repo to download and view them.</p>

**Comment by Lucretia** [comment]
<p>I can gain access to a file in raw form from a private repo using an anonymous browser window. This should not be possible. I have never logged into gh from an anon window.</p>

**Comment by chrispat** [comment]
<p>The only way I can think of this working would be to use the browser tool to capture the time limited SAS URL for the log in question and then use that URL in your anonymous browser.  However, to get that SAS URL you would have needed to be logged into GitHub with access to the private repository.</p>
<p>Is there another route you are taking?  Feel free to email me.</p>

**Comment by Lucretia** [comment]
<p>I’m not asking <em>how</em> to do this, I’m saying <em>I can</em> and I shouldn’t be able to.</p>

**Comment by chrispat** [comment]
<p>And I am asking how you are doing it.  I am not able to find a way outside of grabbing the time bound SAS url.</p>

**Comment by Lucretia** [comment]
<p>Get the raw url from your private repo, go to a private window, paste it in.</p>

**Comment by chrispat** [comment]
<p>Ok yes, that is exactly what I said.  You have to be logged in and have access to the repo to get that signed URL to download the raw log but once you have that URL you can download the raw log for 1 minute.</p>
